components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -818,6 +853,77 @@ public static String getUserConsentURL(OAuth2Parameters params, String loggedInU
         return consentPage;
     }
 
+    private static boolean isAuthEndpointRedirectParamsConfigAvailable() {
+        return FileBasedConfigurationBuilder.getInstance().isAuthEndpointRedirectParamsConfigAvailable();
+    }
+
+    /**
+     * Returns the consent page URL with filtered query params.
+     *
+     * @param redirectUrl    The redirect URL.
+     * @param endpointParams The map for store filtered params.
+     * @param key            The key parameter to refers the filtered params.
+     * @return redirect URL with filtered query params except the key.
+     */
+    public static String getRedirectURLWithFilteredParams(String redirectUrl,
```

Comment:
Are we reusing this method anywhere else?
Should this be public?

Reply:
Resolved by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2020/commits/a4780e126be8e00ac7e209a2553485da861c8cfb